src/psqlgraph/mocks.py

```diff
@@ -1,565 +1,8 @@
-        :type relation: String
-        :return: Boolean
-        """
-        if label in self.relation_cache:
-            return relation in self.relation_cache[label]
-
-        parent_links = self.dictionary.schema[label]["links"]
-
-        links = set()
-        for parent_link in parent_links:
-            if "subgroup" not in parent_link:
-                links.add(parent_link.get("name"))
-                continue
-
-            links |= {link.get("name") for link in parent_link["subgroup"]}
-
-        self.relation_cache[label] = links
-
-        return relation in links
-
-    def make_association(
-        self, src_node: Node, dst_node: Node, edge_label: Optional[str] = None
-    ) -> None:
-        """Create an Edge between 2 nodes
-
-        Given 2 instances of a Node, find appropriate association between the
-        2 nodes and create a relation between them
-
-        There are some special cases like auxiliary_files and
-        structural_variant_calling_workflow, there are 2 different edges between them
-        in opposite directions. In this case, we use the label to differentiate them.
-
-        Bonus: Label could be added to all edges and will make the lookup faster.
-
-        Args:
-            src_node: first node of the edge
-            dst_node: second node of the edge
-            edge_label: label of the edge
-        """
-        if edge_label:
-            edge_class = self.models.Edge.get_subclass(edge_label)
-            if not edge_class:
-                logging.warning(f"Edge with label {edge_label} not found")
-            elif (
-                edge_class.__src_class__ == src_node.__class__.__name__
-                and edge_class.__dst_class__ == dst_node.__class__.__name__
-            ):
-                getattr(src_node, edge_class.__src_dst_assoc__).append(dst_node)
-                return
-            elif (
-                edge_class.__src_class__ == dst_node.__class__.__name__
-                and edge_class.__dst_class__ == src_node.__class__.__name__
-            ):
-                getattr(dst_node, edge_class.__src_dst_assoc__).append(src_node)
-                return
-            else:
-                logging.warning(
-                    "Edge with label {} is not allowed between nodes {} and {}".format(
-                        edge_label, src_node.label, dst_node.label
-                    )
-                )
-
-        link_found = False
-        for assoc_name, assoc_meta in src_node._pg_edges.items():
-            if isinstance(dst_node, assoc_meta["type"]):
-                getattr(src_node, assoc_name).append(dst_node)
-                link_found = True
-                break
-
-        if not link_found:
-            logging.debug(
-                "Could not find a direct relation between '{}'<->'{}'".format(
-                    src_node.label, dst_node.label
-                )
-            )
+from psqlgraph_test_utils.mocks import *
```

Comment:
Is this necessary? Isn't the mock module already being imported? If you find you are needing this, perhaps providing a more formal interface definition in the `__init__.py` would help. Also, I don't think `import *` is recommended by the google style guide.

Reply:
This is temporary for backward compatibility. 
Eventually we want to remove this and everyone should import from new location.

Reply:
I understand why this is being done, essentially to ensure that nothing that uses the original mock will break. My concern is with the implementation itself. Defining the package interface in the __init__.py or providing a specific list of imports would accomplish the same thing while still maintaining adherence to the style guide.

Reply:
updated

Reply:
Update looks good, thank you Qiao!

Also, just an optional suggestion, as part of the testing for this it may make sense to test out this version of psqlgraph pinned to one or more of the dependent projects to verify it's working as expected. If you decide to do this, I noticed that gdcapi has a fixture called `graph_generator` that relies on the mock functionality so running the gdcapi test suite with the feature branch version of psqlgraph pinned would probably be a good test.

Reply:
https://github.com/NCI-GDC/gdcdatamodel2/pull/22/files

Reply:
Nice! Looks good to me, thank you for this!